### ISSUE TITLE
don't repetitively load jquery on staff page

### ISSUE THREAD

**Issue by bufordrat** [uchicago-library/library_website#780]
@vitorgomateus recently noticed that our staff page was loading jQuery 3.3.1 every time it displayed a "schedule an appointment" button.  Oops.

For this issue, please move the jQuery load to the top of the staff page listing.

To see the repetitive jQuery load, navigate here and view source:
https://www.lib.uchicago.edu/about/directory/

**Comment by bbusenius** [comment]
Is this still an issue or was this resolved? I feel like I recall this being resolved.

**Comment by vitorgomateus** [comment]
I checked not so long ago and the issue was still there.

**Comment by bbusenius** [comment]
I see. There's an approved PR outstanding that closes this issue. That's why I thought it was resolved.

**Comment by vitorgomateus** [comment]
That's right. Looks like it was approved but never merged. I'll have to make a change to the navbar so I might do this as well.
